<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerForm</class>
 <widget class="QWidget" name="ServerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableIPadress">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>111</y>
     <width>671</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выходной порт:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>ip-адрес:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0.0.0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="conndisconButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Установить/Разорвать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Соединение:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Таблица клиентов для отправки сообщений</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputPort">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="testButton">
   <property name="geometry">
    <rect>
     <x>544</x>
     <y>80</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Проверить соединение</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>661</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
